### DISCUSSION TITLE
when I make , appearing error: identifier "__hfma2" is undefined . how can I solve

### DISCUSSION THREAD

**Discussion by WangTH37** [NVIDIA/cutlass#1390] [Q&A]
how to solve the problem error: identifier "__hfma2" is undefined

**Comment by jackkosaian** [comment]
Are you building as part of a PyTorch CUDA extension? This comment provides some context on things that can come up with half2 operators when building PyTorch CUDA extensions: https://github.com/NVIDIA/cutlass/issues/746#issuecomment-1426784775
